src/lib/auth/utils.js

```diff
@@ -0,0 +1,6 @@
+// noinspection JSCheckFunctionSignatures
+export const buildUrlEncodedData = obj =>
+    Object.entries(obj).reduce((params, [k, v]) => params.set(k, v.toString()) || params, new URLSearchParams());
```

Comment:
it's a hack using or-shortcutting - an expression-only equivalent to
```js
(params, [k, v]) => { params.set(k, v.toString()); return params; }
```
since params.set returns undefined. it's unclear though, i'll change these instances.